cmd/modelcmd/base.go

```diff
@@ -340,7 +339,7 @@ func (c *CommandBase) doRefreshModels(store jujuclient.ClientStore, controllerNa
 	if err != nil {
 		return errors.Trace(err)
 	}
-	defer modelManager.Close()
+	defer func() { _ = modelManager.Close() }()
```

Comment:
Is this just to expose that we are suppressing the err return? (cause otherwise it does exactly the same thing, right?)

Reply:
yes.